suite-native/module-accounts-import/src/screens/XpubScanScreen.tsx

```diff
@@ -65,7 +65,10 @@ const submitButtonStyle = prepareNativeStyle(utils => ({
 const DEFAULT_CURRENCY_SYMBOL = 'btc';
 // Slip 0014 - https://github.com/satoshilabs/slips/blob/master/slip-0014.md#bitcoin-segwit--p2wpkh--bip84
 const BTC_HARD_CODED_XPUB =
-    'xpub6BiVtCpG9fQPxnPmHXG8PhtzQdWC2Su4qWu6XW9tpWFYhxydCLJGrWBJZ5H6qTAHdPQ7pQhtpjiYZVZARo14qHiay2fvrX996oEP42u8wZy';
+    'zpub6rjNNddoAVvuYaD6WPdxiqFEToQHgrERjWMg7kM9gGGk6rhPMWNEmL5X745FGqBq8Wp136LfA3A7UjRGEYdJrf8dUfshzNrb5rvaryNfVJf';
+
+const TESTNET_HARD_CODED_XPUB =
```

Comment:
Can you please make this some kind of object `Record<NetworkSymbol, Xpub>` then later you can do `devXpubs[selectedNetworkSymbol]`

Reply:
[8ea4764](https://github.com/trezor/trezor-suite/pull/6593/commits/8ea47644d7cd6de42e64047c41dce684fe2fc18d)